executor/include/execution/executor/cuda_executor.hpp

```diff
@@ -55,7 +55,8 @@ struct cuda_executor : executor<cuda_executor, Interface, Cardinality, Blocking,
   // Temporary fix for unit test compilation
   template <typename F>
   void bulk_execute(F f, std::size_t n) const {
-    bulk_execute(f, std::array<int, 6>{1, 1, 1, static_cast<int>(n), 1, 1});
+    bulk_execute(std::forward<F>(f),
```

Comment:
I'm unsure if I was the one suggesting this before. If yes, my apologies. `f` is not a universal reference so calling `std::forward<F>` is a useless I believe. are you interested in a) forwarding or b) copying?
if a)
```cpp
  template <typename F>
  void bulk_execute(F&& f, std::size_t n) const;
```
if b) then use `std::move(f)` to pass it to bulk_execute

Reply:
The intention was to forward but for some reason, the kernel doesn't work with value reference. Will remove forward for now until can figure out the cause of the issue.

Reply:
https://stackoverflow.com/questions/7257144